sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -392,6 +393,11 @@ public abstract partial class ClientOptions
         [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
         public override string? ToString() { throw null; }
     }
+    public partial class ClientRedirectsOptions
+    {
+        public ClientRedirectsOptions() { }
```

Comment:
Super nit: `ClientRedirectOptions` ?

Reply:
I had the same inclination, but I think the plural is intentional.

Reply:
Why is that?